### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by okaycj**
# Summary
This PR contains the test cases for the fix in  #1482.

### CODE REVIEW

File: studies/test_recording_method.py [hunk 1]
```diff
@@ -135,44 +151,63 @@ def test_external_researcher_preview(self):
...
-@skip("Need AWS credentials.")
 class RecordingMethodEFPTestCase(TestCase):
     def test_efp(self):
         study_type = StudyType.get_ember_frame_player()
-        user, study, child = get_user(study_type)
+        _, study, child = get_user(study_type)
         response = Response.objects.create(
             study_type=study_type, child=child, study=study
         )
         self.assertEqual(response.recording_method, "pipe")
```
Comment (becky-gilbert) [inline studies/test_recording_method.py:162]: For the EFP test case, I was going to suggest that we add tests for the web/external view and researcher preview view (like you've done with external and jsPsych study types. But then I realized that the EFP recording method comes from the response POST, so I assume that's why we don't need a test for it here?

**Review by becky-gilbert — APPROVED** [review]
Fantastic, thank you!

File: studies/test_recording_method.py [hunk 2]
```diff
@@ -1,15 +1,15 @@
 import datetime
 import uuid
-from unittest import skip
-from urllib.parse import parse_qs, urlparse
+from datetime import date
```
[the remark this answers is not in the document]
Comment (okaycj) [reply studies/test_recording_method.py:3]: No reason.  I copied it from Python documentation. They're equal.

File: studies/test_recording_method.py (hunk 1 above)
Comment (okaycj) [reply studies/test_recording_method.py:162]: "Need" is a strong word. I did this to be complete, and the only way I could confirm that EFP was setting the recording method correctly.  The inference is that if EFP uses the API to generate the Response and  EFP relies on the default of the `recording_method` field, then this makes sure that the default setting remains "pipe."

Comment (okaycj) [reply studies/test_recording_method.py:162]: Testing for how EFP creates and uses the response should be in the EFP test cases.

**Pull request merged**